lib/Runtime/Base/ThreadContext.cpp

```diff
@@ -3230,6 +3230,53 @@ ThreadContext::InvalidateProtoInlineCaches(Js::PropertyId propertyId)
     }
 }
 
+void
+ThreadContext::InvalidateMissingPropertyInlineCaches(const Js::Type *type, Js::PropertyId propertyId) {
```

Comment:
> { [](start = 101, length = 2)

nit: `{` goes on next line (same comment for `if (inlineCacheList->Empty()) {`)

Reply:
Fixed